src/engine/Story.ts

```diff
@@ -387,16 +431,59 @@ export class Story extends InkObject {
       }
 
       this.state.didSafeExit = false;
+      this._sawLookaheadUnsafeFunctionAfterNewline = false;
 
       if (this._recursiveContinueCount == 1)
         this._state.variablesState.batchObservingVariableChanges = false;
 
       this._asyncContinueActive = false;
+      if (this.onDidContinue !== null) this.onDidContinue;
```

Comment:
I think this should be calling onDidContinue, eg `this.onDidContinue()`

Reply:
Good catch!